README.md

```diff
@@ -2,11 +2,11 @@
 # CrowdStrike OAuth API
 
 Publisher: Splunk  
-Connector Version: 4.2.0  
+Connector Version: 4.3.0  
 Product Vendor: CrowdStrike  
 Product Name: CrowdStrike  
 Product Version Supported (regex): ".\*"  
-Minimum Product Version: 6.1.1  
+Minimum Product Version: 6.2.1  
```

Comment:
```suggestion
Minimum Product Version: 6.2.2 
```

Reply:
There is a customer waiting for this change and I didn't want to risk them not being able to upgrade. Will update this on the next release.